jfrog-cli/artifactory/cli.go

```diff
@@ -2350,3 +2377,9 @@ func createBuildAddGitConfiguration(c *cli.Context) (buildAddGitConfiguration *b
 	buildAddGitConfiguration.ConfigFilePath = c.String("config")
 	return
 }
+
+func extractCurlCommand(c *cli.Context) (command []string) {
```

Comment:
Since this function is based on the way code-gangsta parses command, I think that this is the right place for it. I also use a copy of the Args[] as we tend to not change the original command in our code.